manifests/unit_file.pp

```diff
@@ -73,7 +73,7 @@
   Stdlib::Absolutepath                     $path      = '/etc/systemd/system',
   Optional[Variant[String, Sensitive[String], Deferred]] $content = undef,
   Optional[String]                         $source    = undef,
-  Optional[Stdlib::Absolutepath]           $target    = undef,
+  Optional[String]                         $target    = undef,
```

Comment:
Empty string isn't valid here, right?

```suggestion
  Optional[String[1]]                      $target    = undef,
```

Reply:
Could it be reduced to  `Enum[Stdlib::Absolutepath,Systemd::Unit,Systemd::Unit]` or really need things like `../sshd.service` for instance.

Reply:
> Could it be reduced to Enum[Stdlib::Absolutepath,Systemd::Unit,Systemd::Unit] or really need things like ../sshd.service for instance.

that would work for me, i don't think there's a case for ../